spec/yaml/swiftnav/sbp/ssr.yaml

```
@@ -210,9 +211,11 @@ definitions:
         desc: STEC residual
     - stddev:
         type: u8
+        units: TECU
         desc: >
-          Modified DF389. class 3 MSB, value 5 LSB.
-          stddev = (3^class * (1 + value/16) - 1) * 10
+          Encoded standard deviation leaning on RTCM DF389 specification.
```

Question: not sure about the English wording here. Would the following not be clearer?
```suggestion
          Encoded standard deviation similar to RTCM DF389 specification.
```